src/wazuh_modules/vulnerability_scanner/src/scanOrchestrator/scanContext.hpp

```diff
@@ -846,7 +849,7 @@ struct TScanContext final
      *
      */
     bool m_isInventoryEmpty = false;
-
+    // LCOV_EXCL_STOP
```

Comment:
This LCOV_EXCL_STOP has no corresponding LCOV_EXCL_START

Reply:
I just reverted the changes here https://github.com/wazuh/wazuh/pull/21404/files#diff-8795e9a926870216a18b327045918099ac6752cfc0c1fc46bd792661f05063ed